src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRenderer.php

```diff
@@ -62,8 +62,8 @@ public function render($exception, string $format = 'html'): string
             throw new ErrorRendererNotFoundException(sprintf('No error renderer found for format "%s".', $format));
         }
 
-        if (!$exception instanceof FlattenException) {
-            $exception = FlattenException::create($exception);
+        if ($exception instanceof \Throwable) {
```

Comment:
the signature of these methods in this new component should be changed to `\Throwable|FlattenException`

Reply:
indeed, done